engine/orchestration/src/org/apache/cloudstack/engine/orchestration/VolumeOrchestrator.java

```diff
@@ -1423,7 +1423,7 @@ public void prepare(VirtualMachineProfile vm, DeployDestination dest) throws Sto
                     }
                 }
                 catch (Throwable e){
-                    s_logger.debug("[NSX_PLUGIN_LOG] ERROR: " + e.getMessage());
```

Comment:
@nvazquez thanks for the work!
Sorry, but I have to ask, do we need this try/catch block?
It seems that the code at 1413-1423 does not throw checked exceptions. Is there some kind of runtime exception that might happen in between 1413-1423 that we want to "hide"/swallow?

Reply:
The log level should be moved to `warn`or `error` so that the message get caught in the logs when not running in debug. Putting error message as debug should only be done if they are irrelevant.

Reply:
@marcaurele, do you think we need this try/catch? If we need, what about a more specific one, what exception we want to catch here? The way it is, we are catching much more than Pokemons here...

Reply:
Thanks @rafaelweingartner @marcaurele, sorry for late response.
@rafaelweingartner good observation, I think we can either remove try/catch block or throw a checked exception which will be caught on `VirtualMachineManagerImpl#orchestrateStart`, what do you think?
@marcaurele you're right, done, thanks

Reply:
I would remove the try/catch block. I really did not seen any relevant exception in that piece of code to be dealt if there.

Reply:
Agree with @rafaelweingartner - it should be removed all together. It would be hard anyway to track the problem since the log output will only output the message and not the stack trace. If we're trying to catch any error, the stack is very important to understand the issue. This would normally be done at a higher level if the exception isn't caught here.